test-integration/helpers/window-position.coffee

```diff
@@ -0,0 +1,12 @@
+windowPosition = (test) ->
```

Comment:
It's not so much the state that bothers me with (long) closures, it's the possibility for errors.  

With a closure you've got this "magic" variable(s) that each function has to take care not to modify.  It's pretty easy to accidentally create a variable with the same name as the closure value in a function and accidentally alter the closure value for all the other functions.  In a short 1/2 screen long block that's not so bad but when you have a longer one it's easy to forget that `test` is a reserved var.

In the object based setup the state is indicated by either a `@` or `this.`  That makes it readily apparent that that var is special and it won't be alter if you create a plain var with the same name.

I also suspect that the prototype based method will perform a bit better than re-creating an object each time, but that's speculation and it's quite possible the new fangled optimizers will treat each one the same.